src/ucp/core/ucp_ep.inl

```diff
@@ -44,9 +44,14 @@ static inline ucp_lane_index_t ucp_ep_get_tag_lane(ucp_ep_h ep)
     return ucp_ep_config(ep)->key.tag_lane;
 }
 
-static inline int ucp_ep_is_rndv_lane_present(ucp_ep_h ep)
+static inline int ucp_ep_is_rndv_lane_present(ucp_ep_h ep, ucp_lane_index_t idx)
 {
-    return ucp_ep_config(ep)->key.rndv_lane != UCP_NULL_LANE;
+    return ucp_ep_config(ep)->key.rndv_lanes[idx] != UCP_NULL_LANE;
+}
+
+static inline int ucp_ep_rndv_num_lanes(ucp_ep_h ep)
```

Comment:
better use UCS_F_ALWAYS_INLINE

Reply:
all other funcs there are static inline
will add separate PR to set UCS_F_ALWAYS_INLINE